moveit_ros/visualization/motion_planning_rviz_plugin/src/ui/motion_planning_rviz_plugin_frame.ui

```diff
@@ -641,7 +641,7 @@
                <double>0.010000000000000</double>
               </property>
               <property name="value">
-               <double>1.000000000000000</double>
+               <double>0.100000000000000</double>
```

Comment:
I suggest to revert these changes and leave the ui untouched.

Reply:
Fine with me, but I also think it can't hurt to apply it in both places.